### PR TITLE
Change/Add starting clothes in Bhikkhu profession

### PR DESCRIPTION
**Pull request by Tykato**
#### Summary

SUMMARY: Content "Change/Add starting clothes in Bhikkhu profession"

#### Purpose of change

I contributed the Bhikkhu profession to the game a few days ago.
Now i added some additional clothing.

#### Describe the solution

I added additional clothing. Changed the wear order of the clothes aswell.

#### Describe alternatives you've considered

Leaving them as is.

#### Testing

1. Started up the game with a new Bhikkhu character.
2. Everything worked as intended.

#### Additional context

The sprites for the new clothing items are being done for UltiCa in the upcoming days.

### CODE REVIEW

**Comment by actual-nh** [comment]
I thought you wrote they didn't wear pants?

**Comment by Tykato** [comment]
I'm still changing the clothing to the correct pants, but i accidentally already sent the request. I'm new to github. Give me a moment to figure it out. :)

**Comment by Tykato** [comment]
Okay, here we go! Instead of the Kasaya (the whole three-part robe), the profession now wears the correct three parts (antarvasa (pants), uttarasanga (shirt) and samghati (cloak)).

**Comment by Tykato** [comment]
To clear up some of the confusion: Bhikkhu's don't wear pants, but they wear a traditional three-part robe that has a pants-like-skirt (the antarvasa). I added the correct clothing and removed the pants from the load order.

File: data/json/items/armor/legs_clothes.json [hunk 1]
```diff
@@ -1,4 +1,32 @@
 [
+  {
+    "id": "antarvasa",
+    "repairs_like": "pants",
+    "type": "ARMOR",
+    "name": { "str": "antarvasa" },
+    "description": "A traditional, ankle-length Buddhist skirt. The bottom protrudes, and appears in the rough shape of a triangle.",
```
Comment (LaVeyanFiend) [inline data/json/items/armor/legs_clothes.json:7]:
```suggestion
    "description": "A traditional, ankle-length Buddhist skirt.  The bottom protrudes, and appears in the rough shape of a triangle.",
```

**Comment by Rivet-the-Zombie** [comment]
Do any of the new clothing bits added to the Bhikkhu actually spawn anywhere in-world?

**Comment by Tykato** [comment]
I'm not sure how item spawning in the world works, so i'm not sure if they spawn anywhere. Someone with more experience please help!

**Comment by LaVeyanFiend** [comment]
These will not spawn anywhere until you add them to an itemgroup. Add them to a group in data/json/itemgroups/Clothing_Gear/clothing.json as ``[ "[id of clothing]", [spawn weight] ]``.

**Comment by Tykato** [comment]
> 
> 
> These will not spawn anywhere until you add them to an itemgroup. Add them to a group in data/json/itemgroups/Clothing_Gear/clothing.json as `[ "[id of clothing]", [spawn weight] ]`.

I added it!

**Comment by actual-nh** [comment]
@Tykato: What are you trying to do? (Your last two commits seem not to be doing anything.)

**Comment by Tykato** [comment]
> 
> 
> @Tykato: What are you trying to do? (Your last two commits seem not to be doing anything.)

Different checks seem to randomly fail. I'm trying to figure out why github does this, as a re-upload seems to trigger the checks anew and makes a different checker fail. Didn't know someone would get a notice! Oops!

**Comment by actual-nh** [comment]
The problem was due to a spacing error in a different JSON file that got fixed more recently than the Travis checks ran.

**Comment by anothersimulacrum** [comment]
I've kicked Travis, no need to do anything.

**Pull request merged**